logback-classic/src/main/java/ch/qos/logback/classic/spi/PackagingDataCalculator.java

```diff
@@ -181,13 +174,28 @@ String getCodeLocation(Class type) {
         return "na";
     }
 
-    private String getCodeLocation(String locationStr, char separator) {
-        int idx = locationStr.lastIndexOf(separator);
+    String getCodeLocation(String locationStr) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/tony19/logback/file/2371174401/issues/source?bid=3222618&fileBranchId=3222660#l177)